src/neubot/dash_impl.hpp

```diff
@@ -87,12 +87,12 @@ static inline void loop_request(Var<Transport> transport, int speed_kbit,
 
                     if (time_elapsed < 0) {
                         logger -> warn("Time elapsed can't be negative");
-                        cb(error, nullptr);
+                        cb(ValueError(), nullptr);
```

Comment:
@bassosimone I thought that this is the most appropriate error to use. If you think that a new error would be more accurate tell me the name to use for it.